discord-scripts/thread-management/auto-join.ts

```diff
@@ -23,13 +23,15 @@ import {
 // it to mention the right role. Discord's behavior in this scenario is not to
 // ping the role, but to add all its members to the thread.
 
-const CUSTOM_CHANNEL_ROLE: Record<string, string> = {
-  // hiring: "PeopleOps",
-  "biz-dev-investor": "BD",
-  "press-relations": "M Group",
+interface ChannelRoleMapping {
```

Comment:
Not blocking but let's fix in a follow-up: as a general rule, use `type * = {` instead of `interface * {`.

```suggestion
type ChannelRoleMapping = {
```